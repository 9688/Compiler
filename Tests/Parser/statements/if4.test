if(x){
   if(y){
      if(x);
   }
}