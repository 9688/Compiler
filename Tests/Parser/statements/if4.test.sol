
IF
     list
          x
     BLOCK
          IF
               list
                    y
               BLOCK
                    IF
                         list
                              x
                         list

