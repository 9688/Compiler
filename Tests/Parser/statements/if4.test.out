└──''
Ошибка синтаксиса в строке 4 позиции 4: требуется выражение

Ошибка синтаксиса в строке 5 позиции 2: требуется "}"


